### ISSUE TITLE
black screen 

### ISSUE THREAD

**Issue by cheese666** [wownero/cyberwow#11]
after the hard fork all I get is a black screen, nothing ever happens. does not even try to connect. I have triedfdroid, play store and here.

**Comment by fuwa0529** [comment]
Yeah, I can confirm it. Probably the new bundle format missed the binary, will update soon.

**Comment by fuwa0529** [comment]
A new version should be up soon on play :p

**Comment by cheese666** [comment]
okay. I just uninstall, then found the folder and deleted every trace. it is now pulling up and running but downloading 215061 blocks. I hope it takes them all and does not freeze on the last ones from the hard fork like another coin did, I won't say the name but it was a viking God lol, that took what felt like forever to fix. 

this will take a long long time but I'll keep you all in the loop if it works.

**Comment by fuwa0529** [comment]
What? You fixed it without the update?

**Comment by bomb-on** [comment]
WOW!

**Comment by fuwa0529** [comment]
@cheese666 Can I close this now?
